packages/web3-shared/evm/constants/getContractOwner.ts

```diff
@@ -0,0 +1,77 @@
+import Aave from '@masknet/web3-constants/evm/aave.json'
+import Airdrop from '@masknet/web3-constants/evm/airdrop.json'
+import ArtBlocks from '@masknet/web3-constants/evm/artblocks.json'
+import CryptoArtAI from '@masknet/web3-constants/evm/cryptoartai.json'
+import Gitcoin from '@masknet/web3-constants/evm/gitcoin.json'
+import GoodGhosting from '@masknet/web3-constants/evm/good-ghosting.json'
+import ITO from '@masknet/web3-constants/evm/ito.json'
+import Lido from '@masknet/web3-constants/evm/lido.json'
+import MaskBox from '@masknet/web3-constants/evm/mask-box.json'
+import NftRedPacket from '@masknet/web3-constants/evm/nft-red-packet.json'
+import PoolTogether from '@masknet/web3-constants/evm/pooltogether.json'
+import RedPacket from '@masknet/web3-constants/evm/red-packet.json'
+import Trader from '@masknet/web3-constants/evm/trader.json'
+import { filter, flatten, pick, uniq, values } from 'lodash-unified'
+
+const collect = <T extends Record<string, Record<string, string | number>>>(data: T, fields: (keyof T)[]): string[] => {
+    const groupedByFields = values(pick(data, fields))
+    const listOfAddresses = groupedByFields.map((v) => values(v))
+    const listOfAddress = flatten(listOfAddresses)
+    const collected = filter(uniq(listOfAddress), Boolean) as string[]
+    return collected.map((addr) => addr.toLowerCase())
+}
+
+const domainAddressMap: Record<string, string[]> = {
+    'uniswap.org': collect(Trader, [
```

Comment:
Will these links use for opening in the browser? Lost protocol.

Reply:
No, there is no such requirement yet.